script/Ciel/Day 7/07_01_CIEL07_7.txt

```diff
@@ -335,27 +338,29 @@ And this is what he once said: If you ever happen to meet my younger brother, pl
 [sha:463c3bc1d1b36c44bae3d13ffd9df6ad0cb3401c]{
 -- Page 605, Offset 16262. Glued.
 -- なんだそれみっともねえ！
- What the fuck, how absolutely asinine!
+ What the fuck, how dare he!
```

Comment:
This sounds all wrong to me. Mario would never say this, etc.

Reply:
Hm, I felt the same way about 'asinine'. 
> Fuck's sake, that guy's a disgrace!

?

Reply:
Sounds good